roles/reproducer/tasks/configure_controller.yml

```diff
@@ -469,20 +453,6 @@
             owner: zuul
             group: zuul
 
-        - name: Inject crc related host entry
```

Comment:
We may want to split this out to a separate change? Looks like it is causing issues in the reproducer molecure job?

Reply:
yup, I'm testing another PR to fix that - I'll update this one shortly.